firebase-inappmessaging/src/main/java/com/google/firebase/inappmessaging/internal/ImpressionStorageClient.java

```diff
@@ -96,4 +98,38 @@ private void clearInMemCache() {
         .map(CampaignImpression::getCampaignId)
         .contains(campaignId);
   }
+
+  /** */
+  public void clearImpressions(FetchEligibleCampaignsResponse response) {
+    ArrayList<String> idsToClear = new ArrayList<>();
+
+    for (CampaignProto.ThickContent content : response.getMessagesList()) {
+      String id =
+          content.getPayloadCase().equals(CampaignProto.ThickContent.PayloadCase.VANILLA_PAYLOAD)
+              ? content.getVanillaPayload().getCampaignId()
+              : content.getExperimentalPayload().getCampaignId();
+      idsToClear.add(id);
+    }
+    Logging.logd("Clearing impressions for: " + idsToClear.toString());
+    getAllImpressions()
+        .defaultIfEmpty(EMPTY_IMPRESSIONS)
+        .flatMapCompletable(
+            (storedImpressions) -> {
+              Logging.logd("Existing impressions: " + storedImpressions.toString());
```

Comment:
I think these logs are for debugging? if not, lets remove this (here and below)

Reply:
I think we can leave these logs as debug?